src/phoenix/server/email/password_reset.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
I would put this in a templates directory for cleanliness

Reply:
OK